guides/micronaut-integration-testing/groovy/src/main/groovy/example/micronaut/HelloRepository.groovy

```diff
@@ -0,0 +1,14 @@
+package example.micronaut
+
+import io.micronaut.context.annotation.DefaultImplementation
+
+@DefaultImplementation(HelloRepositoryDefault.class) // <1>
+interface HelloRepository {
+
+    String getHelloInLanguage(String language)
+
+    void putHelloInLanguage(String language, String hello)
```

Comment:
Add Constraints annotations. e.g. `@NotBlank`. I will rename the method to : 
`save` which is closer to the naming convention used in Micronaut Data repositories.